lte/gateway/deploy/magma_5g_agw/roles/magma-agw/tasks/config5g.yaml

```diff
@@ -0,0 +1,62 @@
+---
+- name: Wait for MME PID 
+  ansible.builtin.shell: pidof mme
+  register: result
+  until: result.stdout | length > 0
+  retries: 50
+  delay: 10
+
+- name: Disable stateless
+  shell: magmad_cli.py config_stateless disable
+  when: magma_5g_upgrade
+
+- name: Copy mme yml file
+  copy:
+    src: roles/magma-agw/files/mme.yml
+    dest: "/var/opt/magma/configs/mme.yml"
+  register: mme_yml_file
```

Comment:
can you use process config files present at /etc/magma/ and `sed` required changes.

Reply:
Yes, we can process but as per the developer's it's always good to overried the values by putting the config file inside `/var/opt/magma/configs`. It will give you two benefits: 1. Configuration's will be remain same even after a package upgrade. 2. By removing the configurations you can disable the 5G mode. If you still want to edit the configuration of /etc/magma then I will work on it.